### ISSUE TITLE
OpenSSL compilation on AIX doesn't "just work" with OpenXL 7.1.2

### ISSUE THREAD

**Issue by pshipton** [eclipse-openj9/openj9#20207]
```
15:06:19  .ibm-clang: error: unknown argument '-qpic'; did you mean '-fpic'?
15:06:19  .ibm-clang: error: unknown argument: '-q64'
15:06:19  .ibm-clang: error: unknown argument: '-qmaxmem=16384'
15:06:19  .ibm-clang: error: unknown argument: '-qro'
15:06:19  .ibm-clang: error: unknown argument: '-qroconst'
15:06:19  .ibm-clang: error: unknown argument: '-qthreaded'
15:06:19  gmake[5]: *** [Makefile:4299: apps/lib/libapps-lib-app_libctx.o] Error 1
15:06:19  gmake[4]: *** [Makefile:3254: build_sw] Error 2
15:06:19  gmake[3]: *** [/home/jenkins/workspace/Build_JDK23_ppc64_aix_Personal/closed/openssl.gmk:103: build_openssl] Error 2
15:06:19  gmake[2]: *** [/home/jenkins/workspace/Build_JDK23_ppc64_aix_Personal/closed/custom/Main.gmk:47: j9vm-build] Error 2
```

`OPENSSL_TARGET := aix64-cc` is incorrect. We'll need to determine which compiler is being used to set the target.

**Comment by pshipton** [comment]
There is an `aix64-clang` target, I'll give it a try.

**Comment by pshipton** [comment]
@keithc-ca @jasonkatonica FYI, we'll need to fix this to have builds with OpenXL 17.1.
For details of my build see https://github.com/eclipse-openj9/openj9/pull/20045#issuecomment-2364614158

**Comment by pshipton** [comment]
Fixed via https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/853 and backports.
